admin-guide.md

```diff
@@ -270,3 +277,19 @@ will be captured. Then `"metrics_calculate"` will be called with the
 `"output_metrics"` as the first argument and the file generated in the previous
 step as the second argument. If it exits with a non-zero return code, then the
 test is allowed to pass.
+
+## Testing Víðarr Locally
+
+To test Víðarr locally, ensure that you satisfy these requirements:
+
+- Make sure the .sh file created from the instructions provided in the [Creating a Development Environment](https://github.com/oicr-gsi/vidarr/blob/master/admin-guide.md#creating-a-development-environment) step points to the jars you created in the [Installing Víðarr on a Linux Instance](https://github.com/oicr-gsi/vidarr/blob/master/admin-guide.md#installing-v%C3%AD%C3%B0arr-on-a-linux-instance) step.
+- Use the JSON config file generated during that step as the configuration argument in your command.
+  - Update the `"url"` block to a URL representing your environment, and the `"cromwellUrl"` block to your cromwell server's REST API.
+- Use the [JSON test file](https://github.com/oicr-gsi/empty/blob/master/vidarrtest-regression.json.in) from the [empty](https://github.com/oicr-gsi/empty) repository as the test argument in your command.
```

Comment:
Using empty really isn't a _requirement_, and this instruction is a little incongruous since the regression test file is the first time it's mentioned. 'Testing Workflows' above already describes how to compose a regression test file. empty can be provided as an example though